src/components/Lookup/__tests__/Lookup.spec.js

```diff
@@ -144,10 +144,9 @@ describe('<Lookup />', () => {
 
   it('renders selections correctly', () => {
     mounted.setState({ selected: [sampleData[0]] });
-
     const firstData = sampleData[0];
     const result = mounted.find('.slds-pill');
-    expect(result.find('.slds-pill__remove').length).toBe(1);
+    expect(result.find('button.slds-pill__remove').length).toBe(1);
```

Comment:
Why were these failing?  `expect(result.find('.slds-pill__remove').length).toBe(1)` should not have failed by the looks of it.

Reply:
there are now several elements that have the `slds-pill__remove` class: `Button`, `ButtonIcon` and `button`